scripts/check_changelog_fragments.sh

```diff
@@ -17,7 +17,7 @@ if [ ! -d "${CHANGELOG_DIR}" ]; then
 fi
 
 # diff-filter=A lists only added files
-FRAGMENTS=$(git diff --name-only --diff-filter=A --merge-base origin/master ${CHANGELOG_DIR})
+FRAGMENTS=$(git diff --name-only --diff-filter=A --merge-base "${CHANGELOG_MERGE_BASE:-origin/master}" ${CHANGELOG_DIR})
```

Comment:
Tiny nit: 
```suggestion
FRAGMENTS=$(git diff --name-only --diff-filter=A --merge-base "${MERGE_BASE:-origin/master}" ${CHANGELOG_DIR})
```
(and above)